src/consts.hpp

```diff
@@ -9,7 +9,7 @@ namespace gpudrive {
 namespace consts {
 
 inline constexpr madrona::CountT kMaxAgentCount = 128;
-inline constexpr madrona::CountT kMaxRoadEntityCount = 6000;
+inline constexpr madrona::CountT kMaxRoadEntityCount = 2000;
```

Comment:
I don't think this should be in this PR?

Reply:
Yeah sorry this is just debug code.